ci/request-reviews/get-reviewers.sh

```diff
@@ -59,10 +61,34 @@ for file in "${touchedFiles[@]}"; do
         fi
         # The first regex match is everything after the @
         entry=${BASH_REMATCH[1]}
-        if [[ "$entry" =~ .*/(.*) ]]; then
-            # Teams look like $org/$team, where we only need $team for the API
-            # call to request reviews from teams
-            teams[${BASH_REMATCH[1]}]=
+
+        if [[ "$entry" =~ (.*)/(.*) ]]; then
+            # Teams look like $org/$team
+            org=${BASH_REMATCH[1]}
+            team=${BASH_REMATCH[2]}
+
+            # Instead of requesting a review from the team itself,
+            # we request reviews from the individual users.
+            # This is because once somebody from a team reviewed the PR,
+            # the API doesn't expose that the team was already requested for a review,
+            # so we wouldn't be able to avoid rerequesting reviews
+            # without saving some some extra state somewhere
```

Comment:
I do think that saving some state (maybe in a label?) would make this substantially more flexible; the way it is now as a stateless idempotent task is quite clever though.

Reply:
Bringing state into the picture does add a bunch of complexity, I don't think it's worth it just for this, but maybe in the future